### ISSUE TITLE
Remote reader do not support regex filtering in label

### ISSUE THREAD

**Issue by dmitriy-lukyanchikov** [prometheus/prometheus#3401]
**What did you do?**

I run query like this 
node_cpu_load{host_name=~".*"}

**What did you expect to see?**

metrics that stored on influxdb and prometheus local storage

**What did you see instead? Under which circumstances?**

Only metrics that stored locally inside Prometheus DB under its own retention policy

**Environment**

I run Prometheus with configured remote read and wright to influxdb 1.4

* System information:

Ubuntu 16.04.03
Linux 4.4.0-93-generic x86_64

* Prometheus version:

prometheus, version 1.8.0 (branch: HEAD, revision: 3569eef8b1bc062bb5df43181b938277818f365b)
  build user:       root@bd4857492255
  build date:       20171006-22:12:46
  go version:       go1.9.1

* Prometheus configuration file:
```
# my global config
global:
  scrape_interval:     60s # By default, scrape targets every 15 seconds.
  evaluation_interval: 30s # By default, scrape targets every 15 seconds.
  scrape_timeout: 10s
  # scrape_timeout is set to the global default (10s).

  # Attach these labels to any time series or alerts when communicating with
  # external systems (federation, remote storage, Alertmanager).

# Load and evaluate rules in this file every 'evaluation_interval' seconds.

# A scrape configuration containing exactly one endpoint to scrape:
# Here it's Prometheus itself.
scrape_configs:

  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
  # Auto-discovery via consul tags by Registrator
  - job_name: 'federate_lr_metrics_node'
    scrape_interval: 30s

    honor_labels: true
    metrics_path: '/federate'

    params:
      'match[]':
         - '{__name__="node_load1"}'

    static_configs:
      - targets:
        - 'some_server:9090'


# Remote write configuration (for Graphite, OpenTSDB, or InfluxDB).
remote_write:
  - url: "http://localhost:8086/api/v1/prom/write?db=lr_prom"
# Remote read configuration (for InfluxDB only at the moment).
remote_read:
  - url: "http://localhost:8086/api/v1/prom/read?db=lr_prom"


```

**Comment by juliusv** [comment]
@dmitriy-lukyanchikov Thanks for the report. You are saying you are using InfluxDB 1.4, but I don't think that version exists yet? (at least at https://github.com/influxdata/influxdb/releases, it only goes up to 1.3.7).

Are you using a version of InfluxDB that already has a built-in Prometheus remote read/write endpoint, or using an adapter?

**Comment by dmitriy-lukyanchikov** [comment]
@juliusv Hello! Its version that not released yet but you can build a dcoker image using this repository https://github.com/influxdata/influxdata-docker/tree/master/influxdb/nightly or use oficial site and download version named nightly, also in the link that you provide there is already 1.4 version https://github.com/influxdata/influxdb/releases/tag/v1.4.0

This version of InfluxDB already has built-in Prometheus remote read/write endpoint

**Comment by juliusv** [comment]
@dmitriy-lukyanchikov interesting, https://github.com/influxdata/influxdb/releases/tag/v1.4.0 just gives me a 404, and that tag also does not show for me in the list.

But I guess just checking with nightly would be sufficient.

**Comment by dmitriy-lukyanchikov** [comment]
hm strange indeed the error, ok try to use docker it works perfect for me, it builds in a minute without any problem

**Comment by AlekSi** [comment]
Can it be reproduced with released InfluxDB 1.4.x?

**Comment by brian-brazil** [comment]
Looks like this was resolved, and it would be an issue in InfluxDB rather than Prometheus anyway.
